configure.ac

```diff
@@ -1989,7 +1989,7 @@ if test "$enable_f77" = "yes" ; then
 	# that turns off checking for *everything* is not something that
 	# configure should do - if the user wants this, they can follow
 	# the instructions in the following error message.
-	AC_MSG_ERROR([The Fortran compiler $F77 does not accept programs that call the same routine with arguments of different types without the option $addarg.  Rerun configure with FFLAGS=$addarg])
+        AC_MSG_ERROR([The Fortran compiler $F77 does not accept programs that call the same routine with arguments of different types without the option $addarg.  Rerun configure with FCLAGS=$addarg])
```

Comment:
What's up with the indentation?

Reply:
the commit hook didn't like the tab that was there previously

Reply:
```suggestion
        AC_MSG_ERROR([The Fortran compiler $F77 does not accept programs that call the same routine with arguments of different types without the option $addarg.  Rerun configure with FCFLAGS=$addarg])
```

Reply:
Oh, also `s/$F77/$FC/`

Reply:
thanks for the keen eyes